packages/api/src/errors.js

```diff
@@ -77,3 +77,65 @@ export class InvalidCidError extends Error {
   }
 }
 InvalidCidError.CODE = 'ERROR_INVALID_CID'
+
+export class PinningServiceApiError extends Error {
+  /**
+   *
+   * @param {string} reason
+   * @param {string} [details]
+   * @param {number} [status]
+   */
+  constructor (reason = 'PSA_ERROR', details, status = 400) {
```

Comment:
status is the second argument in HTTPError, can we have it be in the same position here?